code-organization.md

```diff
@@ -237,3 +238,61 @@ But what it gives us is extreme consistency by the most correct thing
 also the easiest.
 
 For a summary, please scroll back up and see the TL;DR section.
+
+## Inheritance
+
+We use a functional approach to inheritance, in this style:
+```
+var animal = function(options) {
+  var self = {},
+      location = 0,
+      speed = 5;
+  self.name = options.name;
+  
+  self.run = function(time) {
+    self.location += time * speed;
+  };
+  
+  self.getLocation = function() {
+    return self.location;
+  }
+  
+  return self;
+};
+
+var bear = animal({ name: 'Oso' });
+bear.run();
+// bear.name => "Oso"
+// bear.getLocation() => 5
+// bear.location => undefined
+
+var bird = function(options) {
+  var self = animal(options);
+  
+  self.fly = function(time) {
+    // Flying is fast
+    self.run(time);
+    self.run(time);
+  };
+  
+  return self;
+};
+
+var duck = animal({ name: 'Pato' });
```

Comment:
should this say `var duck = bird({ name: 'Pato' })`? I think `duck.fly()` below will fail.

...and we all know ducks don't run, they waddle 😝 (but please do not change this example on that account though, I like it as it is)

Reply:
> ...and we all know ducks don't run, they waddle

false.

![image](https://media.giphy.com/media/e2CIuhhEz7nJ6/giphy.gif)

Reply:
lol